cmd/build/helmify/replacements.go

```diff
@@ -66,4 +66,10 @@ var replacements = map[string]string{
         {{- range .Values.controllerManager.exemptNamespaces}}
         - --exempt-namespace={{ . }}
         {{- end }}`,
+	`HELMSUBSET_PDB_POLICY_GROUP_VERSION: ""
```

Comment:
This could break if the ordering of fields in the Kustomize output is not maintained.

It might be better to modify https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/main.go to directly detect these and make the replacement?

Reply:
@maxsmythe I followed the documentation below and made some changes outside of `main.go`, but there is certainly a danger that this implementation will not work.

> please edit kustomization.yaml, kustomize-for-helm.yaml and replacements.go under that directory and then run make manifests.

https://github.com/open-policy-agent/gatekeeper/blob/e00262b76acc82f5dc2a7e3a371d57894e7776b3/charts/gatekeeper/README.md#contributing-changes

I will modify `main.go` to directly detect these. Thanks.